docs/setup.asciidoc

```diff
@@ -1,33 +1,56 @@
 [[setup]]
-== Getting Kibana Up and Running
+== Installing Kibana
```

Comment:
Thank you for changing this...it's bugged me ever since the getting started tutorial was added. :-)